claudia-lab/app/scss/main.scss

```diff
@@ -0,0 +1,13 @@
+@import "_vendor";
+@import "_theme";
```

Comment:
@loomnugget no need to keep the prepended underscore - this gets implied when importing partials:
```
@import "vendor"
@import "theme"
```